docs/visual-testing/integrations/webdriverio.md

```diff
@@ -45,23 +56,38 @@ export const config: Options.Testrunner = {
     })
 ```
 
-- Configure with your Sauce credentials from https://app.saucelabs.com/user-settings.
+### Step 4: Configure your Sauce Labs credentials
+
+Sauce Labs Visual relies on environment variables for authentications.\
+Both `SAUCE_USERNAME` and `SAUCE_ACCESS_KEY` need to be set prior starting your WebdriverIO job.
+
+Username and Access Key car be retrieved from https://app.saucelabs.com/user-settings.
```

Comment:
Username and access key can be retrieved from https://app.saucelabs.com/user-settings.